tasmota/tasmota_xsns_sensor/xsns_62_esp32_mi_ble.ino

```diff
@@ -2985,6 +3006,15 @@ void MI32GetOneSensorJson(int slot, int hidename){
         ResponseAppend_P(PSTR(",\"Btn\":%d"),p->Btn);
       }
     }
+    if (p->feature.flooding){
+      if(p->eventType.flooding || !MI32.mode.triggeredTele || MI32.option.allwaysAggregate
+#ifdef USE_HOME_ASSISTANT
```

Comment:
This change?

Reply:
This logic was copied from line 3000 onwards, for the button input as used by the door sensor. If that doesn't make sense, I can simplify it?